src/cmd/main.go

```
@@ -249,84 +249,91 @@
 // Check for the need of updates if AutoCheckUpdate is on, if its the first time
 // that version is checked or if has more than 24h since the last version check,
 // look into the repo if  there's any more recent version
-// Todo : This is too big of a function. Refactor it to displayUpdateNotification, fetchLatestVersion,
-// shouldCheckForUpdates, chucks
-func CheckForUpdates() {
-	if !common.Config.AutoCheckUpdate {
+func CheckForUpdates(){
```

golangci-lint (1.64.8)

File is not properly formatted

(goimports)